tests/meta/demultiplex/test_demux_post_processing.py

```diff
@@ -485,46 +499,6 @@ def test_finish_all_flowcells(
     assert f"Check demultiplexed flow cell {bcl2fastq_flow_cell.full_name}" in caplog.text
 
 
-def test_add_flow_cell_data_to_housekeeper(demultiplex_context: CGConfig):
```

Comment:
Removed this test. This function just calls other functions that are invoked, these are tested.